samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs

```diff
@@ -392,5 +391,26 @@ protected override async Task OnReactionsRemovedAsync(IList<MessageReaction> mes
                 await turnContext.SendActivityAsync(replyActivity, cancellationToken);
             }
         }
+
+        // This method is invoked when message sent by user is updated in chat.
+        protected override async Task OnTeamsMessageEditAsync(ITurnContext<IMessageUpdateActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var replyActivity = MessageFactory.Text("Message is updated");
+            await turnContext.SendActivityAsync(replyActivity, cancellationToken);
+        }
+
+        // This method is invoked when message sent by user is undeleted/restored in chat.
```

Comment:
Is undeleted and restored here referring to the same action? Could you remove "restored" to avoid confusion?

Reply:
yes removed it